### PR TITLE
Clarify language around resuming closed AudioContexts.

### LINKED ISSUE #1100: Section 2.1.7 unclear about resuming a closed context
In https://webaudio.github.io/web-audio-api/#system-resources-associated-with-an-audiocontext, the third paragraph says

> suspend and close allow authors to release system resources. Releasing system resources means releasing the system resources such as threads, processes, audio streams, but conserving the state of the AudioContext such that it can continue to operate later after resuming if needed.

The last sentence make no restrictions on resuming, but a closed AudioContext cannot be resumed.  We should make this explicit somehow.

### PR DESCRIPTION
**Pull request by joeberkovitz**
Fix #1100

### CODE REVIEW

**Comment by joeberkovitz** [comment]
@rtoy @padenot PTAL

File: index.html [hunk 1]
```diff
@@ -1526,35 +1526,37 @@ <h3 id="lifetime-AudioContext" class="informative">
...
           <h3>
-            System resources associated with an AudioContext
+            System resources associated with BaseAudioContext subclasses
           </h3>
           <p>
-            <a>AudioContext</a>s should be considered expensive objects.
-            Creating an <a>AudioContext</a> often involves creating a
-            high-priority thread, and using a low-latency system audio stream,
-            both having an impact on energy consumption. Creating more than one
-            <a>AudioContext</a> in a document is most of the time unnecessary.
```
Comment (padenot) [inline index.html]: This PR removes this sentence, is that intentional ?

File: index.html [hunk 2]
```diff
@@ -1526,35 +1526,37 @@ <h3 id="lifetime-AudioContext" class="informative">
...
-            "acquiring">acquiring system resources</dfn>. This means opening a
-            system audio stream. This operation returns when the audio stream
-            is ready.
+            authors to <dfn id="releasing">release system resources</dfn>,
+            including threads, processes and audio streams. Suspending an
+            <a>BaseAudioContext</a> releases some of its resources, and allows
+            it to continue to operate later by invoking <a href=
+            "#widl-BaseAudioContext-resume-Promise-void">resume</a>. Closing an
+            <a>AudioContext</a> completely releases all of its resources, after
+            which it cannot be used or resumed again.
```
Comment (padenot) [inline index.html]: It's allowed to release resources, not mandatory. I'd put something like:

> Closing an AudioContext allows implementations to release resources

to put an emphasis on the fact that this is an implementation choice.

**Comment by joeberkovitz** [comment]
@rtoy reminder

File: index.html (hunk 1 above)
Comment (joeberkovitz) [inline index.html]: The removal was intentional (the language felt like overstepping) , but perhaps I overdid it. WIll reinstate.

File: index.html [hunk 3]
```diff
@@ -1526,35 +1526,40 @@ <h3 id="lifetime-AudioContext" class="informative">
...
+            should be considered expensive objects. Creating these objects may
+            involve creating a high-priority thread, and using a low-latency
+            system audio stream, both having an impact on energy consumption.
+            It is usually not necessary to create more than one
+            <a>AudioContext</a> in a document.
+          </p>
+          <p>
+            Constructing or resuming a <a>BaseAudioContext</a> subclass
+            involves <dfn id="acquiring">acquiring system resources</dfn>. This
+            means creating a high-priority thread and (for <a>AudioContext</a>
```
Comment (rtoy) [inline index.html]: "high-priority thread" isn't required.

File: index.html [hunk 4]
```diff
@@ -1526,35 +1526,40 @@ <h3 id="lifetime-AudioContext" class="informative">
...
+            only) creating a system audio stream. This operation returns when
+            the audio stream is ready.
```
Comment (rtoy) [inline index.html]: Do we have a concept of "ready" really means?  If so, a link to this would be great.

File: index.html [hunk 5]
```diff
@@ -1526,35 +1526,40 @@ <h3 id="lifetime-AudioContext" class="informative">
...
-            conserving the state of the <a>AudioContext</a> such that it can
-            continue to operate later after resuming if needed.
-          </p>
-          <p>
-            Constructing or resuming an <a>AudioContext</a> involves <dfn id=
-            "acquiring">acquiring system resources</dfn>. This means opening a
-            system audio stream. This operation returns when the audio stream
-            is ready.
+            authors to <dfn id="releasing">release system resources</dfn>,
+            including threads, processes and audio streams. Suspending an
```
Comment (rtoy) [inline index.html]: "Suspending an" -> "Suspending a"

**Pull request merged**